src/open-systems.jl

```diff
@@ -89,8 +91,8 @@ function liou2kraus( l::Matrix )
   choi2kraus( liou2choi( l ) )
 end
 
-function kraus2choi{T}( k::Vector{Matrix{T}} )
-  c = zeros(T,map(x->x^2,size(k[1])))
+function kraus2choi( k::Vector )
```

Comment:
Here is one case at least where I prefer the parametric method, because it guarantees that `k` has a uniform element type.

Reply:
Would your comment extend to all the functions previously defined parametrically?  Forcing all matrices or vectors to have uniform entries?

Reply:
Would this be for type safety ? I  agree. If things like `Eye` and other `AbstractMatrix`s become more common in the future, you'll get a method error here and can loosen the restriction to `AbstractMatrix{T}` and still enforce a uniform element type.